nncf/ptq/openvino/quantization.py

```diff
@@ -0,0 +1,73 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from typing import Optional
+
+import openvino
+from openvino.tools import pot
+
+from nncf.ptq.data.dataloader import NNCFDataLoader
+from nncf.ptq.openvino.dataloader import POTDataLoader
+
+
+def quantize_impl(model: openvino.runtime.Model,
+                  calibration_dataset: NNCFDataLoader,
+                  preset: str,
+                  target_device: str,
+                  subset_size: int,
+                  fast_error_correction: bool,
+                  model_type: Optional[str] = None) -> openvino.runtime.Model:
+    """
+    Implementation of the `quantize()` method for the OpenVINO backend.
+    """
+    ir_model_xml = '/tmp/model.xml'
```

Comment:
Would be great to avoid hardcoded paths. One possible alternative - [tempfile](https://docs.python.org/3.6/library/tempfile.html#module-tempfile)

Reply:
Thanks! That's a good point. I fixed it.